tests/integration/test_remoted/test_multi_agents/test_multi_agents_reconnection.py

```diff
@@ -0,0 +1,141 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+
+type: integration
+
+brief: This test module aims to validate multiple agents can reconnect at the same without any problem,
+the expected functionality is to all the agents are reconnected correctly and there is no WARNING being
+raised.
+
+components:
+    - remoted
+
+suite: test_multi_agents_reconnection
+
+targets:
+    - manager
+
+daemons:
+    - wazuh-remoted
+
+os_platform:
+    - Linux
+
+os_version:
+    - Centos 8
+    - Ubuntu Focal
```

Comment:
Add the correct OS.

Reply:
Added on commit [`c26b877`](https://github.com/wazuh/wazuh-qa/pull/3662/commits/c26b8771fc6813796e737b7414f958c70dceae60)